### PR TITLE
OLM Sync

### PR DESCRIPTION
**Pull request by slaskawi**
## Additional Information

Please merge this one after those two PRs are in:

* https://github.com/operator-framework/community-operators/pull/1478
* https://github.com/operator-framework/community-operators/pull/1479

## Checklist:
- [ ] Verified by team member
- [ ] Comments where necessary
- [ ] Automated Tests
- [ ] Documentation changes if necessary

### CODE REVIEW

**Comment by slaskawi** [comment]
@mhajas May I ask for a review? This is purely sync with the Operatorhub. Nothing special here. 

It's also worth to mention, this manifests will become a starting point for the RHSSO Operator manifests.

File: deploy/olm-catalog/keycloak-operator/9.0.2/keycloakbackups.keycloak.org.crd.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+                    Opaque     stringData:       GPG_PUBLIC_KEY: <GPG Public Key>
+                    \      GPG_TRUST_MODEL: <GPG Trust Model>       GPG_RECIPIENT:
+                    <GPG Recipient> \n For more information, please refer to the Operator
+                    documentation."
+                  type: string
+                schedule:
+                  description: If specified, it will be used as a schedule for creating
+                    a CronJob
+                  type: string
+              type: object
```
Comment (mhajas) [inline deploy/olm-catalog/keycloak-operator/9.0.2/keycloakbackups.keycloak.org.crd.yaml:62]: Not sure whether this is an issue but on this place in deploy/crds/... we have instanceSelect. Should these two files be synced?

Comment (slaskawi) [reply deploy/olm-catalog/keycloak-operator/9.0.2/keycloakbackups.keycloak.org.crd.yaml:62]: Thanks for spotting this!

**Comment by slaskawi** [comment]
I unified all the file names and added `v` prefix before every CSV (I was asked to do it a while ago). I'm not sure if we are allowed to do such changes, but let's see:

* https://github.com/operator-framework/community-operators/pull/1613
* https://github.com/operator-framework/community-operators/pull/1614

Once those two PRs get in, we can merge this one.

**Comment by slaskawi** [comment]
Removing the `Hold` label as both PRs against the Operatorhub have been merged.

**Pull request merged**